support-frontend/assets/pages/digital-subscription-landing/components/paymentSelection/helpers/paymentSelection.jsx

```diff
@@ -31,21 +31,41 @@ const BILLING_PERIOD = {
   [Monthly]: {
     title: 'Monthly',
     singlePeriod: 'month',
-    salesCopy: (displayPrice: string, saving?: string) => (
-      <span>14 day free trial, then <strong>{displayPrice}</strong> a month for 12 months {saving && null}
-        <br className="product-option__full-screen-break" />
-      </span>
+    salesCopy: (displayPrice: string, saving?: string) => ({
+      control: () => (
+        <span>14 day free trial, then <strong>{displayPrice}</strong> {saving && null}
+          <br className="product-option__full-screen-break" />
+          <br className="product-option__full-screen-break" />
+        </span>
+      ),
+      variantB: () => (
+        <span>
+          14 day free trial, then <strong>{displayPrice}</strong> {saving && null}
+          <br className="product-option__full-screen-break" />
+          <br className="product-option__full-screen-break" />
+        </span>
```

Comment:
I can't tell the difference other than the <span> is on the previous line, is that because differences are stored elsewhere?